andela/ah-orcas-frontend
# #161254628 Implement social sharing

#### What does this PR do?
```This PR implements social sharing via Facebook, Email, Twitter and Facebook ```
#### Description of Task to be completed?
```A user should be able to share/post/tweet articles on social media platforms like Facebook, Email, WhatsApp and Twitter ```
#### How should this be manually tested?
- Clone this repository
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
  `cd ah-orcas-frontend`
- Checkout to the appropriate branch
  `git checkout ft-social-sharing-161254628`
- Install the required dependencies:
   `npm start`
- You should be able to see the social buttons beside every article. Click on the one you want to use to share the article.
#### What are the relevant pivotal tracker stories?
[#161254628](https://www.pivotaltracker.com/story/show/161254628)
#### Screenshots
![image](https://user-images.githubusercontent.com/25586421/50100464-a536dd00-0231-11e9-86ff-2bc1a8ccb7d3.png)


#### Checklist:
